cmd/providers.go

```diff
@@ -28,6 +29,9 @@ func init() {
 	cmdProvidersCreateOneLogin.Flags().StringVar(&subdomain, "subdomain", "", "OneLogin subdomain")
 	cmdProvidersCreateOneLogin.Flags().StringVar(&username, "username", "",
 		"Don't ask for a username and use this instead")
+	cmdProvidersCreateOneLogin.Flags().StringVar(&region, "region", "US",
+		"Region in which onelogin API lives")
```

Comment:
I would change the flag's description to: `Region in which the OneLogin API lives` (match the capitalization of the other flags plus adding "the").

Reply:
Fair point, I'll do that